cmd/lifecycle/main.go

```diff
@@ -19,21 +20,22 @@ func main() {
 		cmd.Exit(err)
 	}
 
+	var platform cmd.Platform = lplatform.NewPlatform(platformAPI)
```

Comment:
```suggestion
	var newPlatform cmd.Platform = lplatform.NewPlatform(platformAPI)
```
It is just a suggestion because I just don't like seeing in the code lines like `platform: platform`.

In addition I have a small question - do we need `cmd.Platform` because otherwise it'll be considered as `*Platform`?

Reply:
`platform := lplatform.NewPlatform(platformAPI)` works too, I'll make this change. 

I'm not sure `newPlatform` adds any useful information vs. `platform` and I like the current terseness so propose keeping it as is for now. Though I could be persuaded if there is another variable name that is more descriptive... we do have a bunch of `thing: thing` in our code so extending this pattern doesn't bother me.